homework_3/task_1.py

```diff
@@ -0,0 +1,11 @@
+my_list = ['Ivan', 'Maria', 'Jane', 'Olaf', 'Margo', 'Nat']
```

Comment:
every HW should be in a separate PR.
ALso why do you use some list with names& while you have data in your task